docs/extensions.adoc

```diff
@@ -899,21 +899,21 @@ class I extends AbstractMethodInterceptor { I(def s) {} }
 
 // On SpecInfo
 specInfo.addSharedInitializerInterceptor new I('shared initializer')
-specInfo.sharedInitializerMethod?.addInterceptor new I('shared initializer method')
+specInfo.specsBottomToTop*.sharedInitializerMethod*.addInterceptor new I('shared initializer method')
 specInfo.addInterceptor new I('specification')
 specInfo.addSetupSpecInterceptor new I('setup spec')
-specInfo.setupSpecMethods*.addInterceptor new I('setup spec method')
+specInfo.specsBottomToTop.collectMany { it.setupSpecMethods }*.addInterceptor new I('setup spec method')
 specInfo.allFeatures*.addInterceptor new I('feature')
 specInfo.addInitializerInterceptor new I('initializer')
-specInfo.initializerMethod?.addInterceptor new I('initializer method')
+specInfo.specsBottomToTop*.initializerMethod*.addInterceptor new I('initializer method')
 specInfo.allFeatures*.addIterationInterceptor new I('iteration')
 specInfo.addSetupInterceptor new I('setup')
-specInfo.setupMethods*.addInterceptor new I('setup method')
+specInfo.specsBottomToTop.collectMany { it.setupMethods }*.addInterceptor new I('setup method')
 specInfo.allFeatures*.featureMethod*.addInterceptor new I('feature method')
 specInfo.addCleanupInterceptor new I('cleanup')
-specInfo.cleanupMethods*.addInterceptor new I('cleanup method')
+specInfo.specsBottomToTop.collectMany { it.cleanupMethods }*.addInterceptor new I('cleanup method')
 specInfo.addCleanupSpecInterceptor new I('cleanup spec')
-specInfo.cleanupSpecMethods*.addInterceptor new I('cleanup spec method')
+specInfo.specsBottomToTop.collectMany { it.cleanupSpecMethods }*.addInterceptor new I('cleanup spec method')
 specInfo.allFixtureMethods*.addInterceptor new I('fixture method')
 
 // on FeatureInfo
```

Comment:
I think we should make it more obvious that this is not needed in addition to `SpecInfo`, but as an focused alternative.

When fixing #1635, I initially had just copied both and was confused by duplicated entries, until I realized that the `SpecInfo` already included what is added in the `on FeatureInfo` section 🤦 .